cmd/backup-operator/README.md

```diff
@@ -0,0 +1,91 @@
+# etcd backup operator
+
+## Overview
+
+etcd backup operator backups the data of a etcd cluster running on [Kubernetes][Kube] to a remote storage such as AWS [S3][s3].
+
+## Getting Started
+
+You can try out etcd backup operator by running it on Kubernetes and then create a `Backup` Custom Resource which contains the targeting etcd cluster and S3 backup config; the etcd backup operator automatically picks up the `Backup` Custom Resource, retrieves etcd snapshot, and then saves it as a backup to S3.
+>Note: The demo uses the `default` kubernete namespace.
+
+Prerequisites: 
+* access to a Kubernetes environment. See TODO.
+* a running etcd cluster named `example-etcd-cluster`. See [Instruction][etcd_cluster_deploy].
+
+
+Once `example-etcd-cluster` is running, you can create a backup using etcd backup operator. 
+
+First, deploy an etcd backup operator:
+> Note: etcd backup operator also creates Backup CRD when starting.
+
+```sh
+$ kubectl create -f example/etcd-backup-operator/deployment.yaml
+$ kubectl get po
```

Comment:
$ kubectl get pods
just to keep it consistent with the other docs.
https://github.com/coreos/etcd-operator#create-and-destroy-an-etcd-cluster